Cargo.toml

```diff
@@ -95,6 +95,7 @@ winit = { version = "=0.30.9", features = ["serde"] }
 xdg = "2.5.2"
 notify-debouncer-full = "0.5.0"
 regex = "1.11.1"
+once_cell = "1.21.3"
```

Comment:
`OnceCell` is in the standard
https://doc.rust-lang.org/std/cell/struct.OnceCell.html